models/points.go

```diff
@@ -248,6 +251,20 @@ func ParseKey(buf []byte) (string, Tags, error) {
 	return string(buf[:i]), tags, nil
 }
 
+func ParseTags(buf []byte) (Tags, error) {
+	return parseTags(buf), nil
+}
+
+func ParseName(buf []byte) ([]byte, error) {
+	// Ignore the error because scanMeasurement returns "missing fields" which we ignore
+	// when just parsing a key
+	state, i, err := scanMeasurement(buf, 0)
+	if state == tagKeyState {
+		return buf[:i-1], nil
+	}
+	return buf[:i], err
+}
```

Comment:
The comment says "ignore the error" but it gets passed back. I just want to make sure that's intentional.